doc/api/deprecations.md

```diff
@@ -2699,6 +2699,19 @@ resolutions not in `node_modules`. This means there will not be deprecation
 warnings for `"exports"` in dependencies. With `--pending-deprecation`, a
 runtime warning results no matter where the `"exports"` usage occurs.
 
+<a id="DEP0XXX"></a>
+### DEP0XXX: `message.aborted`
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/36641
+    description: Documentation-only deprecation.
+-->
+
+Type: Documentation-only
+
+Use `message.destroyed` as [`http.IncomingMessage`][] extends {stream.Readable}.
```

Comment:
Could you add an explanation on why this is deprecated? Also this should specify if `message.aborted` and `message.destroyed` are always equivalent.

Reply:
The deprecation related commit is moved to https://github.com/nodejs/node/pull/36670